layer/utils/network.ts

```
@@ -113,6 +113,8 @@ export const getNetworkFromAddress = (address: string): Network => {
       return Network.Fetch
     case address.startsWith('mantra'):
       return Network.Mantra
+    case address.startsWith('xion'):
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)